libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -1608,10 +1620,10 @@ void GCS_MAVLINK::send_accelcal_vehicle_position(uint32_t position)
   motors. That can be dangerous when a preflight reboot is done with
   the pilot close to the aircraft and can also damage the aircraft
```

Comment:
we should implement it though... probably a reexec. The only thing that would be compatible would probably be the ESC cal